dozer/cogs/tba.py

```diff
@@ -37,100 +41,268 @@ def __init__(self, bot):
     @bot_has_permissions(embed_links=True)
     async def team(self, ctx, team_num: int):
         """Get information on an FRC team by number."""
-        team_data = self.parser.get_team(team_num)
         try:
-            getattr(team_data, "Errors")
-        except tbapi.InvalidKeyError:
-            e = discord.Embed(color=blurple)
-            e.set_author(name='FIRST® Robotics Competition Team {}'.format(team_num),
-                         url='https://www.thebluealliance.com/team/{}'.format(team_num),
-                         icon_url='https://frcavatars.herokuapp.com/get_image?team={}'.format(team_num))
-            e.add_field(name='Name', value=team_data.nickname)
-            e.add_field(name='Rookie Year', value=team_data.rookie_year)
-            e.add_field(name='Location',
-                        value='{0.city}, {0.state_prov} {0.postal_code}, {0.country}'.format(team_data))
-            e.add_field(name='Website', value=team_data.website)
-            e.add_field(name='TBA Link', value='https://www.thebluealliance.com/team/{}'.format(team_num))
-            e.set_footer(text='Triggered by ' + ctx.author.display_name)
-            await ctx.send(embed=e)
-        else:
-            raise BadArgument("Couldn't find data for team {}".format(team_num))
+            team_data = await self.session.team(team_num)
+        except aiotba.http.AioTBAError:
+            raise BadArgument(f"Couldn't find data for team {team_num}.")
+        if team_data.city is None:
```

Comment:
A comment explaining that the only teams with a null city are those that have only a number and a "Team {team number}" name would most likely be useful to future devs.